src/main/java/net/datafaker/providers/base/Aviation.java

```diff
@@ -28,6 +32,14 @@ public String warplane() {
         return resolve("aviation.aircraft.warplane");
     }
 
+    public String general() {
+        return resolve("aviation.aircraft.general");
+    }
+
+    public String transport() {
```

Comment:
Same question, re “airplane”.

Or are these more like “freight” planes?

Again aka aDoc might help for the future.

Reply:
Renamed into "cargo", added JavaDoc.